web/client/components/TOC/Toolbar.jsx

```diff
@@ -282,7 +282,7 @@ class Toolbar extends React.Component {
                             </Button>
                         </OverlayTrigger>
                         : null}
-                    {this.props.activateTool.activateWidgetTool && (status === 'LAYER') && this.props.selectedLayers.length === 1 && !this.props.settings.expanded && !this.props.layerMetadata.expanded && !this.props.wfsdownload.expanded ?
+                    {this.props.activateTool.activateWidgetTool && (status === 'LAYER') && !this.props.selectedLayers.length && !this.props.settings.expanded && !this.props.layerMetadata.expanded && !this.props.wfsdownload.expanded ?
```

Comment:
Can you explain me this change, please?

Reply:
This change is related to the second paragraph of the connected task:
![image](https://user-images.githubusercontent.com/32982343/72425972-e03f7300-3799-11ea-9a9a-9c93c49f0378.png)

Our goal is to check if the length of the `selectedLayers` array is more than zero then we don't want to show the widgets button and we want to show the widgets button for other cases when the length of the array is zero.

**Another possible solution is to remove the code responsible for displaying the widgets buttob at all (in `web/client/components/TOC/Toolbar.jsx`):**

![image](https://user-images.githubusercontent.com/32982343/72426652-34972280-379b-11ea-89f0-173e406338e1.png)

**but I'm not sure if this does not break the logic of other parts of the application.**